tests/bluetooth/tester/src/btp_bap.c

```diff
@@ -541,18 +563,25 @@ static void stream_released(struct bt_bap_stream *stream)
 
 	audio_conn = &connections[a_stream->conn_id];
 
-	if (audio_conn->unicast_group != NULL) {
+	if (stream->dir == BT_AUDIO_DIR_SINK) {
+		/* Stop send timer */
+		k_work_cancel_delayable(&a_stream->audio_clock_work);
+		k_work_cancel_delayable(&a_stream->audio_send_work);
+	}
```

Comment:
The reason for asking, is that `dir` is relative to the ACL role. 

So a `BT_AUDIO_DIR_SINK` stream is a stream to a sink ASE, i.e. `tx` for central/client but `rx` for peripheral/server`

Reply:
I haven't gotten to the streaming tests for server yet. But thanks for pointing this out. I have just checked this with unicast_audio_server app and there is a bug, because for server `stream-dir` is never set, hence always `0`.

Reply:
Or maybe does it mean I have to set `stream-dir` manually in server app layer? Unicast client sets `stream->dir' with `unicast_group_add_stream`.

Reply:
No, `stream-dir` is basically a bug :D It is being removed in https://github.com/zephyrproject-rtos/zephyr/pull/60256